### PR TITLE
Add design process to the handbook

### LINKED ISSUE #8: Add design process to handbook

### PR DESCRIPTION
**Pull request by anabangueses**
Resolves #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,46 @@ Almost everything can be automated through scripts. Process like deploy and dist
 
 Experiences and knowledge must be shared, it's the only way to grow. We love to share what we've learnt so far. We can organize talks or workshops designed for the client needs.
 
+## Design process of a digital product
+
+##### Branding
+
+- Research of visual referents and inspiration from different branding examples.
+- Design of several proposals based on the previous research and iteration of them.
+- Creation of a _____ based on the logo.
```
Comment (sergigracia) [inline README.md]: `_____` ?

File: README.md [hunk 2]
```diff
@@ -70,6 +70,46 @@ Almost everything can be automated through scripts. Process like deploy and dist
...
+## Design process of a digital product
+
+##### Branding
+
+- Research of visual referents and inspiration from different branding examples.
+- Design of several proposals based on the previous research and iteration of them.
+- Creation of a _____ based on the logo.
+- Definition of brand colors.
+- Selection of the brand font family.
+- Creation of different applications for the branding elements (social networks profiles, business cards, merchandising...).
```
Comment (sergigracia) [inline README.md]: If this is a process and has an order I would use `1.   2.   3. ` instead of bullets.

File: README.md [hunk 3]
```diff
@@ -70,6 +70,46 @@ Almost everything can be automated through scripts. Process like deploy and dist
 
 Experiences and knowledge must be shared, it's the only way to grow. We love to share what we've learnt so far. We can organize talks or workshops designed for the client needs.
 
+## Design process of a digital product
+
```
Comment (sergigracia) [inline README.md]: small intro?

File: README.md [hunk 4]
```diff
@@ -70,6 +70,46 @@ Almost everything can be automated through scripts. Process like deploy and dist
 
 Experiences and knowledge must be shared, it's the only way to grow. We love to share what we've learnt so far. We can organize talks or workshops designed for the client needs.
 
+## Design process of a digital product
```
Comment (sergigracia) [inline README.md]: In terms of titles structure I would create a `## Design` section and inside this one I'd create the `### Design Process` section. In the future you might want to add more things a part from the process in the Design section.

**Review by sergigracia — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (anabangueses) [reply README.md]: done

**Review by sergigracia — APPROVED** [review]
Awesome, great job!

**Pull request merged**